tools/docker-compose/nginx.vh.default.conf

```diff
@@ -22,8 +22,6 @@ server {
 
     # HSTS (ngx_http_headers_module is required) (15768000 seconds = 6 months)
     add_header Strict-Transport-Security max-age=15768000;
-    add_header Content-Security-Policy "default-src 'self'; connect-src 'self' ws: wss:; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' *.pendo.io; img-src 'self' *.pendo.io data:; report-uri /csp-violation/";
```

Comment:
Why remove the content security policy?

Reply:
The end goal is to kept it, it was just removed for demo purposes.

Reply:
Yep, these changes were necessary to demo but we we don't intend to actually change this file.

One of the patternfly components is violating the CSP and preventing the UI from being served: [patternfly-react-core/src/helpers/utils.js#L116](https://github.com/patternfly/patternfly-react/blob/2263de8361b196f4421a3adcbb01b4b2dd410d4a/packages/react-core/src/helpers/util.ts#L116)

Merging https://github.com/ansible/awx/pull/7068 _should_ clear this up because it will include [this commit](https://github.com/patternfly/patternfly-react/commit/1900655a54680a9cdd13a90db02b6005e1851e05#diff-d3283522647dde0024e18db90fe7b1ad) from patternfly-react, which removes the `new Function` call. I've verified this with some light manual testing locally but we'll also want qe to take a look.